lib/src/element/tile.cpp

```diff
@@ -90,68 +90,94 @@ namespace cycfi::elements
          std::sort(elements.begin(), elements.end(),
             [](layout_info lhs, layout_info rhs){ return lhs.index < rhs.index; });
       }
+
+     template<axis Axis, class TileElement, axis OtherAxis = other(Axis)>
+     [[gnu::always_inline]] inline auto compute_limits(basic_context const& ctx, const TileElement &tile) -> view_limits
```

Comment:
Use CYCFI_FORCE_INLINE (defined in infra/support.hpp).

Reply:
done